### DISCUSSION TITLE
название нового репoзитория

### DISCUSSION THREAD

**Discussion by BondarenkoMarina** [community/community#63701] [New to GitHub]
Добрый день! При введении названия нового репозитория выдает ошибку "что-то пошло не так".  Как исправить?  И еще вопрос - у меня не установлен SSH ключ. Как его установить? На компе пока установлен только гитхаб ( и мне кажется надо его обновить, но не понимаю как). Ранее был установлен Гит, но преподаватель сказал установить VSCod. Помогите пожалуйста разобраться

**Comment by queenofcorgis** [comment]
Hi @BondarenkoMarina , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
